addOns/wappalyzer/src/test/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerJsonParserUnitTest.java

```diff
@@ -50,8 +50,6 @@ public void shouldParseExample() {
         assertEquals(2, app.getScript().size());
         assertEquals(0, app.getMetas().size());
         assertEquals(0, app.getImplies().size());
-        assertEquals(2, app.getDom().size());
-        // app.getDom();
```

Comment:
I just meant the commented line.

Reply:
Made changes in the previous commit itself.